monai/bundle/scripts.py

```diff
@@ -1229,8 +1185,10 @@ def onnx_export(
         copy_model_state(dst=net, src=ckpt if key_in_ckpt_ == "" else ckpt[key_in_ckpt_])
 
     converter_kwargs_.update({"inputs": inputs_, "use_trace": use_trace_})
-    onnx_model = convert_to_onnx(model=net, **converter_kwargs_)
-    onnx.save(onnx_model, filepath_)
+
+    onnx_saver = OnnxSaver()
+
+    _export(convert_to_onnx, onnx_saver.save, net=net, filepath=filepath_, **converter_kwargs_)
```

Comment:
To call `onnx.save` with these changes, the call stack will now have `_export`, `OnnxSaver.save`, and `save_onnx` beforehand. Each of these functions does very little if anything beyond just calling the next. I don't see the value of this added level of indirection.